crates/kitsune_p2p/types/src/bin_types.rs

```diff
@@ -99,6 +101,22 @@ macro_rules! make_kitsune_bin_type {
                     self.0.as_slice()
                 }
             }
+
+            #[cfg(feature = "arbitrary")]
+            impl<'a> arbitrary::Arbitrary<'a> for $name {
+                fn arbitrary(u: &mut arbitrary::Unstructured<'a>) -> arbitrary::Result<Self> {
+                    // FIXME: there is no way to calculate location bytes right now,
+                    //        so we're producing arbitrary bytes in a way that the location
+                    //        DOES NOT match the hash. This needs to change, but we can go
+                    //        forward with this for now.
```

Comment:
There's nothing in kitsune that requires the location to match the base hash.

Holochain wants to enforce that, but it should be okay for kitsune testing to not assume that requirement.